packages/swapper/src/swappers/ChainflipSwapper/swapperApi/getUnsignedEvmTransaction.ts

```diff
@@ -0,0 +1,91 @@
+import { AxiosError } from 'axios'
+import { evm } from '@shapeshiftoss/chain-adapters'
+import { bn } from '@shapeshiftoss/utils'
+import { fromChainId, fromAssetId } from '@shapeshiftoss/caip'
+import { KnownChainIds } from '@shapeshiftoss/types'
+
+import type {
+  EvmTransactionRequest, 
+  GetEvmTradeQuoteInput,
+  GetUnsignedEvmTransactionArgs
+} from '../../../types'
+import { isExecutableTradeQuote } from '../../../utils'
+
+import {
+  chainIdToChainflipNetwork,
+  CHAINFLIP_BAAS_COMMISSION,
+} from '../constants'
+import { ChainflipBaasSwapDepositAddress } from '../models'
+import { chainflipService } from '../utils/chainflipService'
+import { getEvmTxFees } from "../utils/getEvmTxFees";
+
+export const getUnsignedEvmTransaction = async ({
+  chainId,
+  from,
+  tradeQuote,
+  supportsEIP1559,
+  assertGetEvmChainAdapter,
+  config
+}: GetUnsignedEvmTransactionArgs): Promise<EvmTransactionRequest> => {
+  if (!isExecutableTradeQuote(tradeQuote)) throw Error('Unable to execute trade')
+
+  const brokerUrl = config.REACT_APP_CHAINFLIP_API_URL
+  const apiKey = config.REACT_APP_CHAINFLIP_API_KEY
+
+  const step = tradeQuote.steps[0]
+  const sellChainflipChainKey = `${step.sellAsset.symbol.toLowerCase()}.${chainIdToChainflipNetwork[step.sellAsset.chainId as KnownChainIds]}`
+  const buyChainflipChainKey = `${step.buyAsset.symbol.toLowerCase()}.${chainIdToChainflipNetwork[step.buyAsset.chainId as KnownChainIds]}`
+  
+  // Subtract the BaaS fee to end up at the final displayed commissionBps
+  let serviceCommission = parseInt(tradeQuote.affiliateBps) - CHAINFLIP_BAAS_COMMISSION
+  if (serviceCommission < 0)
+    serviceCommission = 0
+  
+  const maybeSwapResponse = await chainflipService.get<ChainflipBaasSwapDepositAddress>(
+    `${brokerUrl}/swap` + 
+      `?apiKey=${apiKey}` + 
+      `&sourceAsset=${sellChainflipChainKey}` +
+      `&destinationAsset=${buyChainflipChainKey}` +
+      `&destinationAddress=${tradeQuote.receiveAddress}` +
+      `&boostFee=10` +
+      // TODO: Calculate minprice based on tradeQuote.slippageTolerancePercentageDecimal, step.sellAmountIncludingProtocolFeesCryptoBaseUnit
+      // `&minimumPrice=` +
+      // `&refundAddress=${from}` +
+      // `&retryDurationInBlocks=10` +
+      `&commissionBps=${serviceCommission}`,
+    
+    // TODO: For DCA swaps we need to add the numberOfChunks/chunkIntervalBlocks parameters
+  )
+  
+  if (maybeSwapResponse.isErr()) {
+    const error = maybeSwapResponse.unwrapErr()
+    const cause = error.cause as AxiosError<any, any>
+    throw Error(cause.response!.data.detail);
+  }
+
+  const { data: swapResponse } = maybeSwapResponse.unwrap()
+  
+  const depositAddress = swapResponse.address!
+  const { assetReference } = fromAssetId(step.sellAsset.assetId)
+  
+  throw Error('not yet implemented')
+  
+  // TODO: Figure out what to do here. Just a transfer() call apparently, but how?
```

Comment:
Precisely! If it's just a good ol' send, you're on the right track with `adapter.buildSendTransaction`. That's what web does too in `handleSend`.

There's a few major pieces you'll need here:

- `chainSpecific.contractAddress`. This one should be used when sending *tokens* vs. native asset and will effectively instruct to build a `transfer()` Tx, see https://github.com/shapeshift/web/blob/9cc496959d3def4c30bd21015fe059e7a52f25be/packages/chain-adapters/src/evm/EvmBaseAdapter.ts#L288
- `to` - the final destinatooor of the send, whether that's a transfer (`to` will be used in `transfer()` calldata) call or good ol' send is abstracted away, should be the recipient! 
 - `value` - how much of native asset/asset is being sent, in crypto base unit. Again, the `EVMBaseAdapter` will abstract this away for us and zero out the actual `value` (native asset value) in case of token sends and use calldata args instead 
 - `wallet` - a full blown `HDWallet` instance passed down
 - `accountNumber` - the accountNumber used to initiate the Tx
 - `chainSpecific.data` - since you're just doing sends without any "memo" of sorts, it should always be undefined i.e EVMBaseAdapter will build ERC-20 data if it's a token send
 - `gasLimit` and other fee params - can be gotten with `estimateFees()`

Will comment inline on each TODO